src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Why we need this?

Reply:
Since there are final classes in TransactionOuterClass (and other Iroha related classes) we need this mockito extension. But actually it seems I forgot to exclude that commit from this PR >_<. Going to upload next PR about tests right after this anyways.